frontend/app/views/fragments/form/submitPayPal.scala.html

```diff
@@ -0,0 +1,66 @@
+@import views.support.Pricing._
```

Comment:
This file duplicates submit.scala.html but will be removed when we finally release PayPal